packages/shared/lib/wallet.ts

```diff
@@ -813,6 +818,32 @@ export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: NodeAut
     })
 }
 
+/**
+ * Displays participation (stake/unstake) notification
+ *
+ * @method displayParticipationNotification
+ *
+ * @param {PendingParticipation} pendingParticipation
+ *
+ * @return void
+ */
+function displayParticipationNotification(pendingParticipation: PendingParticipation): void {
```

Comment:
Nice, does this function work with partial staking flows?

Reply:
It should, but I'll double check.

Reply:
Yeah it does display a notification for partially staked funds.